pkg/test-infra/pkg/tidb/operation.go

```diff
@@ -447,3 +522,41 @@ func getDiscoveryMeta(tc *v1alpha1.TidbCluster) (metav1.ObjectMeta, label.Label)
 	}
 	return objMeta, discoveryLabel
 }
+
+func (t *TidbOps) parseNodeFromPodList(pods *corev1.PodList) []clusterTypes.Node {
+	var nodes []clusterTypes.Node
+	for _, pod := range pods.Items {
+		if strings.Contains(pod.ObjectMeta.Name, "discovery") {
+			continue
+		}
+		nodes = append(nodes, clusterTypes.Node{
+			Namespace: pod.ObjectMeta.Namespace,
+			PodName:   pod.ObjectMeta.Name,
```

Comment:
Need fill the `Component` field here,

https://github.com/pingcap/tipocket/blob/6dba244c5f1eed302f5e2928463eb90ad1051d09/pkg/cluster/k8s.go#L115-L129